common/analysis/lint_waiver_config.h

```diff
@@ -0,0 +1,63 @@
+// Copyright 2017-2020 The Verible Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef VERIBLE_LINT_WAIVER_CONFIG_LEXER_H__
```

Comment:
This this file and its `.cc` be renamed to `lint_waiver_config_lexer` for consistency with the include guard, and clarity that this is a lexer class.